tools/train.py

```diff
@@ -144,6 +144,11 @@ def main():
     logger.info(f'Config:\n{cfg.pretty_text}')
 
     # set random seeds
+    if hasattr(cfg, "seed") and args.seed is None:
```

Comment:
If there are two seeds, we need to prompt the user to use `arg.seed`.

Reply:
thanks @hhaAndroid . 

1. would the  following work?
```python
  if hasattr(cfg, "seed"):
        if args.seed is None:
            args.seed = cfg.seed
        else:
            logger.warning(f'Random seed passed both in config file and as an argument. \
            Please don''t set the seed in the config file if also passing it as an argument. \
            Using random seed passed as an argument : {args.seed}')
```
2. should I do the same for `args.determinism`?

Reply:
@hhaAndroid could you please answer my questions? I'd be happy to close this PR as soon as possible

Reply:
It's best. And you can refer to
```python
    if hasattr(cfg, "seed") and args.seed is not None:
        seed = args.seed
        logger.warning..
    elif args.seed is not None:
        seed=args.seed
    elif hasattr(cfg, "seed"):
        seed=cfg.seed
    else:
        seed=None
```

Reply:
determinism can also be considered.

Reply:
Thanks! I'm a bit busy today and tomorrow, but I'll try to get this done by Sunday.

Reply:
Ok